sdks/python/apache_beam/utils/thread_pool_executor.py

```diff
@@ -134,3 +134,12 @@ def shutdown(self, wait=True):
       if wait:
         for worker in self._workers:
           worker.join()
+
+
+class _SharedUnboundedThreadPoolExecutor(UnboundedThreadPoolExecutor):
```

Comment:
Do we expect `UnboundedThreadPoolExecutor` to be used directly after this? Should it be renamed to `_ UnboundedThreadPoolExecutor` ?

Reply:
People can still use UnboundedThreadPoolExecutor and there could be a usecase around having threads from the pool having meaningful names to simplify debugging so I'm going to leave it as is.